forgerock-auth/src/test/java/org/forgerock/android/auth/FRUserMockTest.java

```diff
@@ -405,7 +405,7 @@ static void setFinalStatic(Field field, Object newValue) throws Exception {
     @Test
     public void testLogout() throws InterruptedException, ExecutionException, IOException, JSONException, ParseException, AuthenticationRequiredException, ApiException {
         frUserHappyPath();
-        Config.getInstance().setSSOModel(mockBroadcastModel);
+        Config.getInstance().setSSOBroadcastModel(mockBroadcastModel);
```

Comment:
I think we can move this to setup and remove the duplicates  

  @Before
    public void setUp() {
        when(mockBroadcastModel.isBroadcastEnabled()).thenReturn(true);
        Config.getInstance().setSSOBroadcastModel(mockBroadcastModel);
    }

Reply:
Moved to setup